build_tools/ci/pad_test.sh

```diff
@@ -19,7 +19,7 @@ cd "$TESTDIR"
 
 python3 -m venv sandbox
 source sandbox/bin/activate
-MLIR_AIE=mlir_aie-0.0.1.2024021121+e6874fb
+MLIR_AIE=mlir_aie-0.0.1.2024021415+7ee44b7
```

Comment:
how do you feel about making this an argument, since we may have multiple scripts, it will be easy to track from ci.yml that way?

Reply:
Yeah we can do that